css/amp.css

```diff
@@ -126,6 +126,33 @@ i-amp-sizer {
   /* TODO(dvoytenko): explore adding here object-fit. */
 }
 
+/**
+ * Makes nodes visually invisible but still accessible to screen-readers.
+ *
+ * This is a public class so page authors can use it for similiar a11y purposes
+ * within their AMP documents on any element.
+ *
+ * This css has been carefully tested to ensure screen-readers can read and
+ * activate (in case of links and buttons) the elements with this class. Please
+ * use caution when changing anything, even seemingly safe ones. For example
+ * changing width from 1 to 0 would prevent TalkBack from activating (clicking)
+ * buttons despite TalkBack reading them just fine.
+ */
+.amp-screen-reader {
+  position: fixed !important;
+  top: 0px !important;
+  left: 0px !important;
+  width: 1px !important;
+  height: 1px !important;
+  opacity: 0 !important;
+  overflow: hidden !important;
+  border: none !important;
+  margin: 0 !important;
+  padding: 0 !important;
+  display: block !important;
```

Comment:
does this matter? once you set positioning-absolute/static i think you dont have to set this